### PR TITLE
Use backend API on frontend

### PR DESCRIPTION
**Pull request by pva701**
## Related issue(s)

https://issues.serokell.io/issue/EDNA-43

#### Related changes (conditional)

- Tests
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [ ] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)

#### Stylistic guide (mandatory)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Commit-and-PR-policy-4cf98e1a910a415d86b5f2491d9af1af).
- [ ] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

**Comment by gromakovsky** [comment]
Can you please also delete legacy `ExperimentalMeasurement` and `uploadExperiment` from the backend code? Or you can ask me and I'll do it myself.

UPD: apparently it's still used to show plots. So disregard this comment, let's remove it later.

File: frontend/src/pages/upload/UploadPage.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const UploadPage: FunctionComponent = (): ReactElement => {
   const methodologiesLoadable = useRecoilValueLoadable(methodologiesAtom);
   const [excelFile, setExcelFile] = useRecoilState(excelFileAtom);
 
+  // TODO reset form field on click Add another
```
Comment (gromakovsky) [inline frontend/src/pages/upload/UploadPage.tsx]: Yeah, I even noticed it during testing, but decided that it's not too critical :)

**Review by gromakovsky — APPROVED** [review]
The last fix works for me and overall this version works pretty well. I'm approving it because we need it quite urgently, but I didn't carefully review logic and I'll be glad if @sashasashasasha151 reviews as well.

**Comment by pva701** [comment]
btw @sashasashasasha151 before you start to pizdet' what the fuck I implemented with request ids, just look here: https://recoiljs.org/docs/guides/asynchronous-data-queries#use-a-request-id

it's idiomatic approach suggested by Recoil
one day we will rewrite to more fine grained updates I guess

**Comment by pva701** [comment]
like to re-request only one project and one target when experiment is uploaded

File: frontend/src/components/CreatableSelect.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default function CreatableSelect<T>({
   return (
     <Select<SelectOption>
       {...props}
-      value={value ? toOption(value) : undefined}
+      value={value ? toOption(value) : null}
```
Comment (sashasashasasha151) [inline frontend/src/components/CreatableSelect.tsx:31]: :worry:

**Review by sashasashasasha151 — APPROVED** [review]

File: frontend/src/components/CreatableSelect.tsx (hunk 2 above)
Comment (pva701) [reply frontend/src/components/CreatableSelect.tsx:31]: oru

**Pull request merged**